src/airport.js

```diff
@@ -1,26 +1,28 @@
 
 class Airport {
   airportPlanes;
+  capacity;
   constructor (typeOfTransport, capacity=20, airportPlanes=[]){
     this.capacity=capacity;
     this.airportPlanes = airportPlanes
     this.typeOfTransport = typeOfTransport
   }
 
   landPlane(plane){
-    if ( plane.constructor.name == this.typeOfTransport.name) {
+    let indexPlaneRemove= this.airportPlanes.findIndex(element => element.id === plane.id)
+    if ( plane.constructor.name == this.typeOfTransport.name && indexPlaneRemove===-1) {
```

Comment:
I've had to use the property "constructor.name" and "name" respectively given that I've decoupled the class airport from the class plane, and I had to make sure the object that wants to land is a plane. I'm not sure this is the best way to do it, though